common/test/src/View/AddNodesTest.cpp

```diff
@@ -30,40 +35,62 @@
 #include <vecmath/vec.h>
 #include <vecmath/vec_io.h>
 
+#include <kdl/overload.h>
+
 #include "Catch2.h"
 
 namespace TrenchBroom {
     namespace View {
-        TEST_CASE_METHOD(MapDocumentTest, "AddNodesTest.updateLinkedGroups") {
-            auto* groupNode = new Model::GroupNode{Model::Group{"group"}};
-            document->addNodes({{document->parentForNodes(), {groupNode}}});
-
-            document->select(groupNode);
-            auto* linkedGroupNode = document->createLinkedDuplicate();
-            document->deselectAll();
-
-            document->select(linkedGroupNode);
-            document->translateObjects(vm::vec3(32.0, 0.0, 0.0));
-            document->deselectAll();
-
-            auto* brushNode = createBrushNode();
-            document->addNodes({{groupNode, {brushNode}}});
-
-            REQUIRE(groupNode->childCount() == 1u);
-            CHECK(linkedGroupNode->childCount() == 1u);
-            
-            auto* linkedBrushNode = dynamic_cast<Model::BrushNode*>(linkedGroupNode->children().front());
-            CHECK(linkedBrushNode != nullptr);
-
-            CHECK(linkedBrushNode->physicalBounds() == brushNode->physicalBounds().transform(linkedGroupNode->group().transformation()));
```

Comment:
This stuff with transformations and physical bounds isn't tested by the test you moved in this diff, so I wouldn't delete this one.

Reply:
See 04adbd6 and ed34073.

Reply:
You are right. 02f510af8 restores the deleted test. Good catch!